client/src/main/java/io/pravega/client/stream/impl/SegmentSelector.java

```diff
@@ -47,6 +48,15 @@
 @RequiredArgsConstructor
 public class SegmentSelector {
 
+    /**
+     * When writing without a routing key, keep writing this many bytes to the same segment before selecting a new one.
+     */
+    private static final long BYTES_PER_SEGMENT = 256 * 1024;
```

Comment:
Can we make this a runtime parameter from a Java property? This will allow users to easily optimize this.